components/feature/downloads/src/main/java/mozilla/components/feature/downloads/db/DownloadDao.kt

```diff
@@ -27,6 +27,9 @@ internal interface DownloadDao {
     @Query("SELECT * FROM downloads ORDER BY created_at DESC")
     fun getDownloads(): Flow<List<DownloadEntity>>
 
+    @Query("SELECT * FROM downloads ORDER BY created_at DESC")
+    suspend fun getDownloadsList(): List<DownloadEntity>
```

Comment:
nit: (unrelated to this change) The `DownloadDao` mentions site permissions :)

Reply:
😅 🤦🏽  fixing it.